.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: fmt
       run: |
         cargo fmt --all -- --check
-        ./scripts/check_json_format.sh
+        ./eng/scripts/check_json_format.sh
```

Comment:
I think we can probably get rid of this check since I believe we're no longer using the json based code generation we previously were.